genie-test-web/build.gradle

```diff
@@ -23,7 +25,9 @@ dependencies {
 }
 
 processResources {
-    filter ReplaceTokens, tokens: [
```

Comment:
This belongs in a different commit... no big deal but just in case you want to fix